cmd/controller/main.go

```diff
@@ -8,14 +8,14 @@ import (
 	"github.com/awslabs/karpenter/pkg/cloudprovider"
 	"github.com/awslabs/karpenter/pkg/cloudprovider/registry"
 	"github.com/awslabs/karpenter/pkg/controllers"
-	"github.com/awslabs/karpenter/pkg/controllers/provisioning/v1alpha1/reallocation"
-
 	"github.com/awslabs/karpenter/pkg/controllers/provisioning/v1alpha1/allocation"
+	"github.com/awslabs/karpenter/pkg/controllers/provisioning/v1alpha1/reallocation"
+	termination "github.com/awslabs/karpenter/pkg/controllers/termination/v1alpha1"
```

Comment:
Consider simply not using v1alpha1 as the controller's package name. We're not running multiple api versions (yet). I think it's unlikely we'll ever support different controllers simultaneously.

Reply:
I'd like to keep the pattern here the same until we have the ability to run multiple api versions like you mention.

Reply:
I'll cut an issue for that to track it.